<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SemiAutoMoveDialog</class>
 <widget class="QDialog" name="SemiAutoMoveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Move &amp; Stretch subtitles</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="text">
        <string>Select two subtitle lines, and correct their start times according to the video. The rest of the subtitles shall be arranged accordingly.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget1"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="hours1">
         <property name="maximum">
          <number>99</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="minutes1">
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="seconds1">
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="milliseconds1">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget2"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="hours2"/>
       </item>
       <item>
        <widget class="QSpinBox" name="minutes2">
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="seconds2">
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="milliseconds2">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SemiAutoMoveDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SemiAutoMoveDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
